tests.html

```diff
@@ -276,7 +276,7 @@ <h2 id="g4-title"></h2>
                             SELECT testname, COUNT(run) as failed
                             FROM Tests
                             JOIN TestRuns ON Tests.run = TestRuns.id
-                            WHERE TestRuns.time > ${since} AND Tests.failed = 1 AND project LIKE '${repo}'
+                            WHERE TestRuns.time > ${since} AND Tests.failed = 1 AND project LIKE '${repo}' AND testname NOT LIKE '%selenium%'
```

Comment:
Can you please add a commit message (or comment) like "The selenium tests have a bug where skipped tests show up as failures. They are deprecated and not very flaky, so not interesting to track."